### PR TITLE
test(authentication app): Write unit tests for the remaining current …

### PR DESCRIPTION
**Pull request by collinewait**
**What does this PR do?** 
   - It tests the current functionality and integrates with Travis CI.

**Description of Tasks to be completed?**
- Testing the exceptions returned by a model and login serialiser in the authentication app
 - Testing the creation of a superuser.

**How should this be manually tested?**
- After cloning the repo, cd into the directory. 
- Run ```pip install -r requirements.txt```
- and finally Run ```python python manage.py test```

**What are the relevant pivotal tracker stories?**

- [159951990](https://www.pivotaltracker.com/story/show/159951990)

**Screenshots**

screen shot 2018-09-04 at 07 53 37

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ install:
   - pip install coveralls
 
 script:
-  - coverage run --source=authors/ manage.py test
+  - coverage run --source=authors/ --omit=authors/wsgi.py,authors/settings/production.py,authors/settings/staging.py manage.py test
```
Comment (reiosantos) [inline .travis.yml]: I think these files can be excluded with the exclude tag. In cases where there are so many files to exclude, you wouldn't need to chain them in that one command

**Review by reiosantos — COMMENTED** [review]
you can please correct that in your travis file

**Comment by huxaiphaer** [comment]
@collinewait , there are conflicts on .travis.yml

File: .travis.yml (hunk 1 above)
Comment (collinewait) [reply .travis.yml]: I resolved it @reiosantos

**Pull request merged**